packages/baggage-log-record-processor/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@opentelemetry/baggage-log-record-processor",
+  "version": "0.0.1",
+  "description": "OpenTelemetry Baggage Log Record Processor for Node.js",
```

Comment:
"for Node.js"? I know you are just copying from baggage-span-processor here. Is there anything Node.js-specific in this package?

Reply:
good point, I was just using the same definition. Let me update it to be more generic